Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html

```diff
@@ -4,43 +4,77 @@
 <f:section name="head">
 	<title><neos:backend.translate id="{title}" /></title>
 
-	<f:if condition="{neos:backend.shouldLoadMinifiedJavascript()}">
-		<link rel="stylesheet" type="text/css"
-			href="{f:uri.resource(path: 'Styles/Main.css', package: 'Neos.Neos')}?bust={neos:backend.cssBuiltVersion()}"/>
-	</f:if>
+	<link
+		rel="stylesheet"
+		type="text/css"
+		href="{f:uri.resource(path: 'Styles/Main.css', package: 'Neos.Neos')}?bust={neos:backend.cssBuiltVersion()}"
+	/>
 
 	<f:if condition="{moduleConfiguration.additionalResources.styleSheets}">
-		<f:for each="{moduleConfiguration.additionalResources.styleSheets}" as="additionalResource">
-			<link rel="stylesheet" href="{f:uri.resource(path: additionalResource)}" />
+		<f:for
+			each="{moduleConfiguration.additionalResources.styleSheets}"
+			as="additionalResource"
+		>
+			<link
+				rel="stylesheet"
+				href="{f:uri.resource(path: additionalResource)}"
+			/>
 		</f:for>
 	</f:if>
 
-	<script src="{f:uri.resource(path: 'Library/jquery/jquery-2.0.3.js')}"></script>
+	<script
+		src="{f:uri.resource(path: 'Library/jquery/jquery-2.0.3.js')}"
```

Comment:
Do we still need jQuery here?

Reply:
As it is not just for the configuration module yes.
But can checkt that because I added jquery in 5.0 to the package.json and therefore the used version should be 3.4 something. Good catch!

Reply:
Ok tried it and the configuration module don't need it but the old bootstrap stuff is using that.
The problem is that the webpack.ProvidePlugin don't expose the jquery to the global scope.

Tried the expose-loader but without success.
But removing the bootstrap stuff is also on the list.